pyvelox/pyvelox.h

```diff
@@ -504,6 +527,56 @@ static void addVectorBindings(
             std::move(baseVector),
             PyVeloxContext::getSingletonInstance().pool());
       });
+
+  m.def(
+      "row_vector",
+      [](std::vector<std::string>& names,
+         std::vector<VectorPtr>& values,
+         const std::optional<py::list>& nullability_list) {
```

Comment:
Is it possible to pass a generator / lambda as the nullability_list, i cant imagine many people will like to put true, false, true etc..
maybe even option for a dict of rowNumber:true/false , with no value being understood as rowNumber:true ?